src/app/reducers/index.ts

```diff
@@ -49,3 +59,47 @@ export const getLayoutState = (state$: Observable<State>) =>
 export const getShowSidenav = compose(
   fromLayout.getShowSidenav, getLayoutState
 );
+
+/**
+ * Server Reducers
+ */
+ export const getServerState = (state$: Observable<State>) =>
+   state$.select(state => state.server);
+
+ export const getSignatureFetched = compose(
```

Comment:
we could go for `'dataArrived' , 'dataSaved', 'dataReady'` , where data = [signature,zhang,etc...]